### DISCUSSION TITLE
2005/05/14/%E6%88%91%E8%A6%81%E5%9B%9E%E4%B8%8A%E6%B5%B7%E5%96%BD

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#28] [Announcements]
# 2005/05/14/%E6%88%91%E8%A6%81%E5%9B%9E%E4%B8%8A%E6%B5%B7%E5%96%BD

未来的两天基本都在平流层里，就不来更新了

https://blog.qizhen.xyz/2005/05/14/%E6%88%91%E8%A6%81%E5%9B%9E%E4%B8%8A%E6%B5%B7%E5%96%BD

**Comment by ruanqizhen** [comment]
静
2005年05月16日 11:45 编辑
已经过了两天拉～还不见人影哦～

回复
Nongjie
2005年05月15日 21:25 编辑
欢迎回家啰!!!!!

回复
Jessica
2005年05月15日 16:34 编辑
回来后能不能说一声，有些问题想请教你。我的MSN SPACE很多功能有问题。谢谢！！！

回复
小＆U
2005年05月15日 13:41 编辑
欢迎回上海~~~~~~~~~~

回复
Herman
2005年05月15日 11:26 编辑
兄弟，有空来深圳，我做东

回复
妤
2005年05月14日 20:28 编辑
你原本在哪里？

回复
Huajun
2005年05月14日 17:16 编辑
两天！那么久！

回复
